### PR TITLE
Hide end of form nav

### PR DESCRIPTION
**Pull request by TylerSheffels**
Refer to: http://manage.dimagi.com/default.asp?154572#858916

This doesn't display the default value, but this appears to be the current method for forms.
image

The App settings page handles this differently, but uses a different framework for settings.  Seemed like beyond the scope of this ticket to try and move the form settings into a similar format as the app settings.

### CODE REVIEW

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by czue** [comment]
build failures look real:
```
ERROR: testBuildApp (corehq.apps.app_manager.tests.test_app_manager.AppManagerTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/tests/test_app_manager.py", line 145, in testBuildApp
    copy = app.make_build()
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/models.py", line 2907, in make_build
    copy.create_jadjar(save=True)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/models.py", line 2802, in create_jadjar
    all_files = self.create_all_files()
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/models.py", line 3268, in create_all_files
    'suite.xml': self.create_suite(),
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/models.py", line 3250, in create_suite
    return suite_xml.SuiteGenerator(self).generate_suite()
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/suite_xml.py", line 645, in generate_suite
    self.post_process(suite)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/suite_xml.py", line 706, in post_process
    self.add_form_workflow(suite)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/suite_xml.py", line 727, in add_form_workflow
    if self.app.enable_post_form_workflow():
TypeError: 'bool' object is not callable
```
```
======================================================================
ERROR: test_suite (corehq.apps.app_manager.tests.test_days_ago_migration.DaysAgoMigrationTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/tests/test_days_ago_migration.py", line 13, in test_suite
    suiteB = self.app.create_suite()
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/models.py", line 3250, in create_suite
    return suite_xml.SuiteGenerator(self).generate_suite()
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/suite_xml.py", line 645, in generate_suite
    self.post_process(suite)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/suite_xml.py", line 706, in post_process
    self.add_form_workflow(suite)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/suite_xml.py", line 727, in add_form_workflow
    if self.app.enable_post_form_workflow():
TypeError: 'bool' object is not callable
```

**Comment by snopoke** [comment]
@TylerSheffels need to fix build

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by snopoke** [comment]
@TylerSheffels tests still failing (different ones this time)

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by TylerSheffels** [comment]
@snopoke I believe the error is unrelated (saw it in other builds), so this should be good to go.

File: corehq/apps/app_manager/feature_support.py [hunk 1]
```diff
@@ -68,3 +68,10 @@ def enable_group_in_field_list(self):
         version 2.16 or higher
         """
         return self._require_minimum_version('2.16')
+
+    @property
```
Comment (TylerSheffels) [inline corehq/apps/app_manager/feature_support.py]: @sravfeyn You thought there was a better way for feature support, this is where Danny pointed me.

Comment (sravfeyn) [inline corehq/apps/app_manager/feature_support.py]: Oh yeah. Found this couple of days ago when I had to version-limit a feature.

Comment (sravfeyn) [inline corehq/apps/app_manager/feature_support.py]: Nice!

**Pull request merged**